src/main/java/org/dataconservancy/cos/osf/client/model/Wiki.java

```diff
@@ -0,0 +1,190 @@
+import com.github.jasminb.jsonapi.annotations.Links;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Type("wikis")
+public class Wiki {
+
+    @Id
+    private String id;
+
+    @Relationship(value = "comments", resolve = true, relType = RelType.RELATED, strategy = ResolutionStrategy.OBJECT)
+    private List<Comment> comments;
+
+    @Links
+    private Map<String, ?> links;
+
+    private String name;
+
+    private String kind;
+
+    private String materialized_path;
+
+    private DateTime date_modified;
+
+    @Relationship(value = "node", resolve = true, relType = RelType.RELATED, strategy = ResolutionStrategy.REF)
+    private String node;
+
+    @Relationship(value = "user", resolve = true, relType = RelType.RELATED, strategy = ResolutionStrategy.REF)
+    private String user;
+
+    private String path;
+
+    private String content_type;
+
+    private Map<String, ?> extra; // for "version"
+
+    private int size;
+
+
+    public String getKind() {
+        return kind;
+    }
+
+    public void setKind(String kind) {
+        this.kind = kind;
+    }
+
+    public String getNode() {
+        return node;
+    }
+
+    public void setNode(String node) {
+        this.node = node;
+    }
+
+    public String getUser() {
+        return user;
+    }
+
+    public void setUser(String user) {
+        this.user = user;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getMaterializedPath() {
+        return materialized_path;
+    }
+
+    public void setMaterializedPath(String materialized_path) {
+        this.materialized_path = materialized_path;
+    }
+
+    public String getDateModified() {
+        if (this.date_modified!=null) {
+            return this.date_modified.toString(DATE_TIME_FORMATTER_ALT);
+        } else {
+            return null;
+        }
+    }
+
+    public void setDateModified(String date_modified) {
+        if (date_modified!=null){
+            this.date_modified = JodaSupport.parseDateTime(date_modified);
+        } else {
+            date_modified=null;
+        }
+    }
+
+    public String getPath() {
+        return path;
+    }
+
+    public void setPath(String path) {
+        this.path = path;
+    }
+
+    public String getContentType() {
+        return content_type;
+    }
+
+    public void setContentType(String content_type) {
+        this.content_type = content_type;
+    }
+
+    public int getSize() {
+        return size;
+    }
+
+    public void setSize(int size) {
+        this.size = size;
+    }
+
+    public Map<String, ?> getLinks() {
+        return links;
+    }
+
+    @JsonProperty("links")
+    public void setLinks(Map<String, ?> links) {
+        this.links = links;
+    }
+
+    public Map<String, ?> getVersion() {
```

Comment:
This needs to be named after the field `extra`, like the others.  So `setExtra` and `getExtra`